.github/workflows/linux_simd.yml

```diff
@@ -269,60 +308,182 @@ jobs:
         fetch-depth: 0
     - uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4.7.0
       with:
-        python-version: ${{ env.PYTHON_VERSION }}
+        python-version: '3.11'
     - uses: ./.github/actions
 
-  use_wheel:
+  sdist:
     needs: [smoke_test]
     runs-on: ubuntu-latest
     if: github.event_name != 'push'
-    env:
-      USE_WHEEL: 1
     steps:
     - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
       with:
         submodules: recursive
         fetch-depth: 0
     - uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4.7.0
       with:
-        python-version: ${{ env.PYTHON_VERSION }}
+        python-version: '3.12'
+    - name: Install gfortran and OpenBLAS (MacPython build)
+      run: |
+        set -xe
+        sudo apt install gfortran libgfortran5
+        target=$(python tools/openblas_support.py)
+        sudo cp -r $target/lib/* /usr/lib
+        sudo cp $target/include/* /usr/include
+    - name: Build a wheel via an sdist
+      run: |
+        python -m build
+        pip install dist/numpy*.whl
+    - name: Install test dependencies
+      run: |
+        pip install -r test_requirements.txt
+    - name: Run test suite
+      run: |
+        cd doc
+        pytest --pyargs numpy -m "not slow"
+
+
+||||||| parent of 1a81895ae6 (CI: split `build_test.yml` into three GHA jobs files):.github/workflows/build_test.yml
```

Comment:
and here

Reply:
Thanks @rgommers I think those bits come from a different backport I was working on, one of the reasons for this one. Fixed now.